content/blogs/newsletter-103.md

```diff
@@ -0,0 +1,121 @@
+---
+title:       "Newsletter 103 - Late 2023 news"
+author:      "GeoB99"
+date:        2023-08-06
+tags:        [ newsletter ]
+---
+
+Salutations from the ReactOS project team! On the previous newsletter we talked about the overall ReactOS releasing progress scheme and development status of the project.
+We are attempting to make an effort on publishing regularly at least 3 newsletters per year, depending on how the development workflow goes.
+In this newsletter we will highlight some of the contributions by project developers and contributors alike as well as future plans and headlines.
```

Comment:
Should this also have a short reference to the announcement we made about hiring Hermès?